src/tests/Microsoft.PowerFx.Interpreter.Tests/SerializationTests.cs

```diff
@@ -106,5 +104,13 @@ public BooleanOptionSet(string name, DisplayNameProvider displayNameProvider)
 
             bool IExternalOptionSet.IsBooleanValued => true;
         }
+
+        private class MyOptionSet : OptionSet, IExternalOptionSet
```

Comment:
why do we need this? Can't the test just use `new OptionSet(name, displayNameProvider)`

Reply:
Good point. Removed.